src/seedu/addressbook/data/person/Address.java

```diff
@@ -8,32 +8,54 @@
  */
 public class Address {
 
-    public static final String EXAMPLE = "123, some street";
-    public static final String MESSAGE_ADDRESS_CONSTRAINTS = "Person addresses can be in any format";
-    public static final String ADDRESS_VALIDATION_REGEX = ".+";
-
+    public static final String EXAMPLE = "123, some street, some unit, some postal code";
+    public static final String MESSAGE_ADDRESS_CONSTRAINTS = "Person addresses to be entered as 123, Pasir Ris Drive 4, #12-12, 123123";
+    
+    public static final int BLOCK_NUMBER_INDEX = 0;
```

Comment:
Good use of constant to replace magic number. However, these constants seems unlikely to be useful outside of this class, i.e. they should be kept private.